# 13 Curve regime score

Does the gap between five-print RWTC returns when the curve z-score is above 1 and when it is below -1 keep its sign after 2019?

Holdout starts 2020-01-01. Verdict: held.

(CL1_MINUS_CL4 minus the train mean) divided by the train population standard deviation. Train mean -0.142 dollars per barrel. Train standard deviation 1.850. Both moments stop at 31 Dec 2019 and are applied to every later print. A positive CL1_MINUS_CL4 means the front NYMEX contract is above the fourth.

In [1]:
import json
from pathlib import Path
studies = json.loads(Path('data/studies.json').read_text())
study = next(item for item in studies['studies'] if item['slug'] == 'curve-z')
print(study['verdict'])
print(study['verdict_line'])
print(study['table']['caption'])
for row in study['table']['rows']:
    print(' | '.join(row))


held
Train gap, z above 1 minus z below -1, is -1.331% (927 high, 996 low). Holdout gap is -5.886% (336 high, 39 low). The holdout kept the train sign.
Five-print RWTC return, high curve z versus low curve z
Train | 927 | 996 | -1.331%
Holdout | 336 | 39 | -5.886%


## Method

The score is a level z-score of the stored spread, not a new curve. EIA's contract history in this snapshot ends where RCLC1 ends.

## Result

Train gap, z above 1 minus z below -1, is -1.331% (927 high, 996 low). Holdout gap is -5.886% (336 high, 39 low). The holdout kept the train sign.